portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -66,10 +67,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     boolean readableComment = false;
     boolean readableUsername = false;
 
-    int limit = 10000;
-    limit -= comment.length();
-
-    if (doesLimitExceeded(limit) == true) {
+    if (doesLimitExceeded(TOTAL_COMMENTS_SIZE_LIMIT - comment.length()) == true) {
```

Comment:
Rename the function to isLimitExceeded to be gramatically correct. :)

Reply:
Right, I will fix :)